CMakeLists.txt

```diff
@@ -318,23 +318,13 @@ if(MSVC)
   add_compile_options(/Oi)
   # Enable buffer security check on Debug, disable otherwise
   add_compile_options($<IF:$<CONFIG:Debug>,/GS,/GS->)
-  # Enforce C++ standard conforming conversion rules to catch possible bugs
-  add_compile_options(/permissive-)
```

Comment:
Is this not needed anymore with CMake on Windows, or was this removed by mistake?